src/helper.py

```diff
@@ -198,3 +241,18 @@ def validate_inputs():
 
     validate_inputs()
     return get_assets_fullname_from_collibra()
+
+
+def get_asset_types_name_from_lineage_json_file(path: str) -> set:
```

Comment:
missing return statement?
would be good to create a small test for this as well by adding a simple lineage.json file in the tests folder?